xml/virt_docs.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-docs">
+ <title>Help and documentation</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  Both &xen; and &kvm; hosts include various sources of information and documentation,
+  many of which are already available your installed system. The various documentation
+  packages provide additional content beyond what is available in the base installation.
+ </para>
+ <variablelist>
+  <varlistentry>
+   <term>Integrated help and man pages for shell commands</term>
+   <listitem>
+    <para>
+     The various virtualization packages provide commands for managing many aspects
+     of the virtualization host. It is not possible or expected to remember all
+     options supported by these commands. Most commands include integrated 'help'
+     subcommands and man pages which describe the command and any supported options.
```

Comment:
should we add an example? I would suggest virsh or/and virt-install:

"For example, to install a guest from an OS installation image, the command virt-install is available.

Type:

$ man virt-install

to get detailed information about the program use and all available options. You can find the section labeled "EXAMPLES" towards the end of the man page, with some useful command line examples.

For a more succinct description of available commands and options, run the program as follows:

$ virt-install --help

For tools that can be used interactively, like the "virsh" command,
in addition to 

$ man virsh

and

$ virsh --help

it is also possible to get help out a specific command in the middle of an interactive session, as such:

$ virsh
Welcome to virsh, the virtualization interactive terminal.

Type:  'help' for help with commands
       'quit' to quit

virsh # help
...(to get the list of commands)...

virsh # help start
... (to show the help specific to the "start" command)

Reply:
At the risk of needing to add more examples for the *many* commands we have, I added some for virt-install. I'd think any reasonable sysadmin knows how to use man pages and --help option.